.gitignore

```diff
@@ -3,6 +3,7 @@
 .nox/
 /build/
 /dist/
+/htmlcov/
```

Comment:
I don't like putting it at root, does not allow to run multiple target pythons in parallel; this path needs to be per nox env different

Reply:
I see, thanks!